src/components/Borrow/WrappedBorrow/WrappedBorrow.tsx

```diff
@@ -0,0 +1,61 @@
+import { Grid } from '@material-ui/core'
+import { ExchangeCollateralTokens, ExchangeSyntheticTokens } from '@selectors/solanaWallet'
+import { Vault } from '@synthetify/sdk/lib/exchange'
+import React from 'react'
+import { ActionBorrow } from '../SwitchBorrow/ActionBorrow'
+import ActionMenuBorrow, { IActionContents } from '../SwitchBorrow/ActionMenuBorrow'
+
+export interface BorrowedPair extends Vault {
+  collateralData: ExchangeCollateralTokens
+  syntheticData: ExchangeSyntheticTokens
+}
+interface IProp {
+  pairs: BorrowedPair[]
+}
+export const WrappedBorrow: React.FC<IProp> = ({ pairs }) => {
+  const [cRatio, setCRatio] = React.useState(100.0)
+  const changeCRatio = (nr: number) => {
+    setCRatio(nr)
+  }
+
+  const actionContents: IActionContents = {
+    borrow: (
+      <ActionBorrow
+        cRatio={cRatio}
+        changeCRatio={changeCRatio}
+        interestRate={24.68}
```

Comment:
why are you hardcoding this data here instead of in story?

Reply:
I don't know exactly what numbers are there yet.